packages/react-await/src/Await.tsx

```diff
@@ -1,8 +1,8 @@
 import type { FunctionComponent } from 'react'
 import { createElement, useMemo } from 'react'
 import { useSyncExternalStore } from 'use-sync-external-store/shim'
-import type { Tuple } from '../types'
-import { hashKey } from '../utils'
+import type { Tuple } from './types'
+import { hashKey } from './utils'
```

Comment:
I think it can be a new PR, Why don't we add utils or package for this modules?

![image](https://github.com/suspensive/react/assets/57122180/2b9f250b-c322-46a0-a89a-f6e343cb6abb)
![image](https://github.com/suspensive/react/assets/57122180/afbf2010-ac85-43e1-85db-e1e160f7cba4)

Reply:
Actually I don't think timing is now. but Let's talk about after this Pull Request.